nexusutils/dataconverter/readers/rii_database/dispersion_functions.py

```diff
@@ -55,23 +67,21 @@ def tabulated(table_input: TableInput):
     Parsing for tabulated dispersions from refractiveindex.info
     """
     index_from = {
-        'tabulated nk': ['Wavelength', 'n', 'k'],
-        'tabulated n': ["Wavelength", "n"],
-        'tabulated k': ["Wavelength", "k"],
+        "tabulated nk": ["Wavelength", "n", "k"],
+        "tabulated n": ["Wavelength", "n"],
+        "tabulated k": ["Wavelength", "k"],
     }
 
     daf = pd.read_table(
```

Comment:
Consider renaming the variable `daf`. I don't know what it stands for. Maybe it's obvious for others.

Reply:
It stands for dataframe. Typically in pandas `df` is used, but this is not to the liking of the linter. But I will rename it to something more descriptive.